features/git-rename-branch/destination_branch_exists_locally.feature

```diff
@@ -0,0 +1,36 @@
+Feature: git rename-branch: errors when the destination branch exists locally
+
+  As a developer trying to rename a branch into an already existing branch
```

Comment:
Does one rename into, or rename to? I think rename to.

`As a developer trying to rename a branch to an already existing branch`